aten/src/ATen/nnapi/nnapi_bind.cpp

```diff
@@ -0,0 +1,197 @@
+#include <vector>
+
+#include <torch/script.h>
+#include <torch/custom_class.h>
+
+#include "nnapi_wrapper.h"
+#include "nnapi_model_loader.h"
+
+
+namespace torch {
+namespace nnapi {
+namespace {
+
+nnapi_wrapper* nnapi;
+nnapi_wrapper* check_nnapi;
+
+void load_platform_library() {
+  static int run_once = [](){
+    nnapi_wrapper_load(&nnapi, &check_nnapi);
+    CAFFE_ENFORCE(nnapi);
+    CAFFE_ENFORCE(nnapi->Model_free);
+    CAFFE_ENFORCE(nnapi->Compilation_free);
+    CAFFE_ENFORCE(nnapi->Execution_free);
```

Comment:
Q. Why do you enforce that only 3 functions exist and not the others?

Reply:
The other functions are all called through the `check_nnapi` interface, which throws if the functions are null.  However, these need to be called from destructors, where throwing is not safe, so I call the raw NNAPI functions directly, so I want to verify early that they are valid.